docs/release-notes.md

```diff
@@ -8,6 +8,19 @@ meta:
     content: Fairwinds Insights, Kubernetes Audit, Kubernetes configuration validation
 ---
 # Release Notes
+
+## 5.3.0
+### Team Management
+We launched a new interface and API for managing teams within your organization. Within an organization,
+you can create multiple teams, each of which has access to a particular set of clusters, namespaces, and repositories.
+Members of those teams can be assigned a specific roles to limit the actions they can take on those objects.
```

Comment:
"..can be assigned a specific **role**..**

Reply:
good catch!